AirSkyBoat/AirSkyBoat
# Jormungand Tuning

**_I affirm:_**


## Please enter a player-facing description
- Jormungand has been tuned to more accurately mimic era (Frank)


## What does this pull request do? (Please be technical)
Tiamat Fixes:
Fixes delay to always be 240 from retail testing (Finally got it). The video used prior to tuning delays was from a private server (oops)

Jorm Tuning:
+ Adds immunities to lua
+ Correct damage taken mods
+ Adds fastcast 
+ Adds matt and refresh
+ Fixes Jorm using spells/skill during Blood Weapon
+ Jorm will now wake up when slept in the air
+ Fixes an issue where the horrid roar x3 was not going off. Moved the logic to weaponskill so we are not constantly checking it onMobFight
+ Fixes TP gain in the air


## Steps to test these changes
Fight Jorm
Test immunities to Bind, Light Sleeps, Paralyze, Silence, Terror
Under 25% when Jorm uses horrid roar it will now use 3 in a row
When Jorm uses Blood Weapon make sure he does not cast spell or use mobskills. Use !tp3000 to force a mobskill



## Special Deployment Considerations
n/a
